deployment/clusterObjectModel/cluster_object_model.py

```diff
@@ -29,24 +29,23 @@
 from ..paiLibrary.common import linux_shell
 from .mainParser import kubernetes as pai_com_kubernetes
 from .mainParser import machine as pai_com_machine
+from .mainParser import layout as pai_com_layout
 
 
 package_directory_com = os.path.dirname(os.path.abspath(__file__))
 
 
-
 class cluster_object_model:
 
     def __init__(self, configuration_path):
         self.logger = logging.getLogger(__name__)
         self.configuration_path = configuration_path
-        self.cluster_configuration = file_handler.load_yaml_config("{0}/cluster-configuration.yaml".format(configuration_path))
         overwirte_service_configuration = file_handler.load_yaml_config("{0}/services-configuration.yaml".format(configuration_path))
```

Comment:
So user still prepare 4 files for this module. What if one file doesn't exits?

Reply:
It will tolerant the non existing file.

Reply:
Refined